harness/Makefile

```diff
@@ -30,15 +30,15 @@ check:
 
 .PHONY: test
 test:
-	pytest -v --runslow --durations=0 tests
+	pytest -v -s --runslow --durations=0 tests
```

Comment:
why is `-s` needed now?

Reply:
Last time we considered making this change test_to_device_warnings had problems and the logger didn't work properly unless we added -s. This time, that appears to be insufficient so I'm actually trying a couple of different things. Although you said the test wasn't that valuable, so next step is to just remove it,.

Reply:
I've removed this change, because even with it PyTest was still hanging on that test. I actually just changed the test to not use a subprocess. There was a comment about how we didn't want to mess up the existing processes logging configuration, but I think simply adding a queue logger is negligible. Happy to revisit...